src/components/timeline/VerticalIndicators.js

```diff
@@ -72,7 +72,7 @@ export class VerticalIndicators extends React.PureComponent<Props> {
           break;
         case 'FirstContentfulPaint':
         case 'FirstContentfulComposite':
-          color = 'var(--green-60)';
+          color = 'var(--magenta-60)';
```

Comment:
much better !

I wonder if we should make them thicker too.

Reply:
Yeah, we can make it thicker too. Let's experiment it in a follow-up.